components/SearchBar.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div class="flex flex-row flex-0 w-128">
+        <div id="search-fields" class="grid-cols-3 mx-4">
+            <div class="search-specialty col-span-1 inline-block w-1/3 py-4">
+                <select class="rounded-l-full rounded-r-none w-full px-1
+                    border-2 border-primary/80 py-1.5 drop-shadow-md text-primary-text bg-secondary-bg/5">
+                    <option value="" class="text-primary-text-muted" disabled selected>
+                        Select a specialty...
+                    </option>
+                    <option value=""></option>
+                    <option :key="index" v-for="(specialty, index) in specialtyDropdownOptions">
+                        {{ specialty.names[0].name }}
+                    </option>
+                </select>
+            </div>
+            <div class="search-location col-span-1 inline-block w-1/3 py-4">
+                <select class="rounded-none w-full px-1
```

Comment:
```suggestion
                <select class="w-full px-1
```

Reply:
This isn't needed since it's the default. The only reason to use this is if a border radius is applied for a smaller breakpoint like mobile.

Reply:
Actually I think was supposed to be rounded-full and I was just testing. The cats for tusearch bar isn't final as it doesn't display on sisterly across browsers. I'm considering if we actually need `vue-select` or if it's easier/more maintainable to just build one.